src/main/java/org/ambraproject/wombat/controller/ArticleController.java

```diff
@@ -864,19 +865,6 @@ private XmlContent getXmlContent(Site site, ArticlePointer articlePointer,
     });
   }
 
-  private String getLinkText(Site site, HttpServletRequest request, String doi) throws IOException {
```

Comment:
always nice to remove dead code

Reply:
Yeah, quick win to improve code coverage ;)